src/components/OrgSettings/ActionItemCategories/Modal/ActionItemCategoryModal.tsx

```
@@ -186,102 +187,99 @@ const CategoryModal: FC<IActionItemCategoryModal> = ({
-                variant="danger"
-                onClick={handleDelete}
-                data-testid="deleteCategoryButton"
-                className="btn btn-danger"
-              >
-                <i className="fa fa-trash me-2" />
-                {tCommon('delete')}
-              </Button>
-            )}
+        {/* Disabled Toggle */}
+        <Form.Group className="d-flex flex-column mb-4">
+          <label>{tCommon('disabled')} </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)